packages/expo/src/Notifications/Notifications.ts

```diff
@@ -438,4 +441,56 @@ export default {
     }
     return ExponentNotifications.setBadgeNumberAsync(number);
   },
+
+  async scheduleNotificationWithCalendarAsync(
+      notification: LocalNotification,
+      options: {
+        year?: number;
+        month?: number;
+        hour?: number;
+        day?: number;
+        minute?: number;
+        second?: number;
+        weekDay?: number;
+        repeat?: boolean;
+      } = {}
+    ): Promise<string> {
+      const areOptionsValid: boolean =
+        (options.month == null || isInRange(options.month,1,12)) &&
+        (options.day == null || isInRange(options.day, 1, 31)) &&
+        (options.hour == null || isInRange(options.hour, 0, 23)) &&
+        (options.minute == null || isInRange(options.minute, 0, 59)) &&
+        (options.second == null || isInRange(options.second, 0, 59)) &&
+        (options.weekDay == null || isInRange(options.weekDay, 1, 7)) &&
+        (options.weekDay == null || options.day == null);
+
+      if (!areOptionsValid) {
+        console.warn('Options in scheduleNotificationWithCalendarAsync call were incorrect!');
+        return "unsuccessful";
```

Comment:
This should throw an error! 💣 In fact, maybe it would make sense to include in the error message what's wrong with the options (this could be eg. shown to the user by notification scheduler app developer). 🤔